### ISSUE TITLE
[BUG] Size property off by 1

### ISSUE THREAD

**Issue by Thanos1716** [Traben-0/Entity_Texture_Features#183]
**Describe the issue**

The size property, for a slime at least must be 1 higher than Optifine in order to match the same sized slime
This probably extends to other entities as well, but I haven't tested any.
This issue may possibly affect random entities in EMF as well.

assets/minecraft/optifine/random/entity/slime.properties

First .properties:
```
skins.1=1
sizes.1=0

skins.2=2
sizes.2=1

skins.3=3
sizes.3=3

skins.4=1
```

Second .properties (sizes incremented by 1):
```
skins.1=1
sizes.1=1

skins.2=2
sizes.2=2

skins.3=3
sizes.3=4

skins.4=1
```

Slime textures for testing
[slime2](https://github.com/Traben-0/Entity_Texture_Features/assets/80975570/a196947a-0886-40f5-b16d-f5f17b189cd9)
[slime3](https://github.com/Traben-0/Entity_Texture_Features/assets/80975570/9a2754e6-8ca9-4b9c-9dae-f3717844e007)

**Screenshots**

Optifine (with 1st .properties):
![2023-07-08_14 17 22](https://github.com/Traben-0/Entity_Texture_Features/assets/80975570/d310412b-8dfc-436f-8625-d7425740eaea)

ETF (with 1st .properties):
![2023-07-08_14 19 21](https://github.com/Traben-0/Entity_Texture_Features/assets/80975570/f4dc8bb4-921b-4095-993d-fd472dec56a2)

ETF with second .properties (incremented by 1):
![2023-07-08_14 19 48](https://github.com/Traben-0/Entity_Texture_Features/assets/80975570/e05bd3a2-c6da-4f77-801d-dd189450b3b7)

**Your setup: (please complete the following information):**

- Minecraft version: 1.20.1
- ModLoader: fabric
- ETF Version: 4.3.3

**Comment by Traben-0** [comment]
This has been identified in my discord and will be fixed in the next version

**Comment by Traben-0** [comment]
resolved in version 4.5.0
